### PR TITLE
fix: fix multiple adapter

### PR DESCRIPTION
**Pull request by nodece**
Fix https://github.com/node-casbin/mongoose-adapter/issues/67

Each adapter should have its own model and schema instead of a global model and schema.

### CODE REVIEW

File: .github/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM mongo:latest
+FROM mongo:5.0.16
```
Comment (nodece) [inline .github/Dockerfile:1]: Fixed the mongo version.

File: test/helpers/helpers.js [hunk 2]
```diff
@@ -29,7 +29,16 @@ async function createEnforcer () {
 }
 
 async function createAdapter (useTransaction = false) {
-  return MongooseAdapter.newAdapter('mongodb://localhost:27001,localhost:27002/casbin?replicaSet=rs0', {
+  return MongooseAdapter.newAdapter('mongodb://localhost:27001,localhost:27002/casbin?replicaSet=rs0', {}, {
```
Comment (nodece) [inline test/helpers/helpers.js:32]: Fix the adapter options.

**Comment by shubhanshusingh** [comment]
I tested this fix's build! It works, perfect!

**Comment by shubhanshusingh** [comment]
@nodece  @Shivansh-yadav13  when can we release this new version?

**Pull request merged**